scenario/scenario.go

```diff
@@ -233,8 +233,15 @@ loop:
 
 		// TODO: Evaluate every rule
 		var pair map[string]interface{}
+		var cache map[string]interface{}
+
 		json.Unmarshal(body, &pair) //	convert []byte to map[string]interface{}
 
+		for k, v := range s.cache { //	add all cache to pair
+			json.Unmarshal(v, &cache)
```

Comment:
cache di declare di luar loop, tapi 
 > `json.Unmarshal(v, &cache)`

di dalam loop

jika 
```go
s.cache = {
  "this": {"a": "a"},
  "that": {"b": "b"}
}

pair["this"] = {"a", "a"}
pair["that"] = {"a": "a", "b": "b"}
```
which is wrong